srcpkgs/PikoPixel.app/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'PikoPixel.app'
+pkgname=PikoPixel.app
+version=1.0.b9e
+revision=1
+archs="x86_64"
```

Comment:
Is there a reason this is limited to only x86_64 with glibc?

Reply:
I mentioned it already and why but I had compilation failures when testing cross-compilation to i686, because it couldn't get libobjc.so.4 or something similarly named. You have any clue why that happened?

Reply:
We don't cross compile for i686.

```
./xbps-src -m masterdir-i686 binary-bootstrap i686
./xbps-src -m masterdir-i686 pkg PikoPixel.app
```

The `archs` variable is used to limit architectures if they really can't run on that architecture.

From my testing `i686` and `x86_64-musl` build fine, for the cases that are cross compiled, just let them fail from missing dependencies.

Reply:
So just remove `archs=`

Reply:
Oh that's neat! Nice to see it working in the end on other arches ^_^

Reply:
I'm gonna bring in the change on my end, I was busy trying to deal with some questions from another person

Reply:
change is in now